hugr-llvm/src/extension/int.rs

```diff
@@ -104,6 +199,24 @@ fn emit_int_op<'c, H: HugrView>(
         IntOpDef::igt_u => emit_icmp(context, args, inkwell::IntPredicate::UGT),
         IntOpDef::ile_u => emit_icmp(context, args, inkwell::IntPredicate::ULE),
         IntOpDef::ige_u => emit_icmp(context, args, inkwell::IntPredicate::UGE),
+        IntOpDef::ixor => emit_custom_binary_op(context, args, |ctx, (lhs, rhs), _| {
```

Comment:
seems a shame not to include `iand` here

Reply:
True! I've added it now